tests/index.spec.js

```diff
@@ -138,7 +146,7 @@ describe('tokenExchangeHandler', () => {
         client_secret: 'invalid_secret',
         grant_type: 'client_credentials',
       });
-    expect(res).to.have.status(400);
+    expect(res).to.have.status(401);
```

Comment:
This was updated to `401` because the error.statusCode is 401: https://github.com/digitalbazaar/oauth2-token-endpoint-handler/blob/main/lib/main.js#L144